packages/api/src/external/carequality/gateway/index.ts

```diff
@@ -52,6 +58,38 @@ export async function gatherXCPDGateways(patient: Patient): Promise<Gateways> {
   };
 }
 
+export function facilitiesWithEpicFilter(
+  cqDirectoryEntryModel: CQDirectoryEntry[],
+  isEpicEnabled: boolean
+): CQDirectoryEntry[] {
+  if (isEpicEnabled) {
+    return cqDirectoryEntryModel;
+  }
+
+  return cqDirectoryEntryModel.filter(entry => !isEpicFacility(entry.id, cqDirectoryEntryModel));
+}
+
+function isEpicFacility(gateway: string, cqDirectoryModel: CQDirectoryEntry[]): boolean {
+  const cqDirectoryGW = cqDirectoryModel.find(entry => entry.id === gateway);
+
+  if (cqDirectoryGW) {
+    const managingOrganizationId = cqDirectoryGW.managingOrganizationId;
+    const isImplementer = managingOrganizationId === cqDirectoryGW.id;
+    const isUnderEpic = managingOrganizationId === EPIC_ID;
+    const isEpic = cqDirectoryGW.id === EPIC_ID;
+
+    if (isUnderEpic || isEpic) {
+      return true;
+    } else if (!managingOrganizationId || isImplementer) {
+      return false;
+    }
+
+    return isEpicFacility(managingOrganizationId, cqDirectoryModel);
+  }
+
+  return false;
```

Comment:
as discussed, all we need here is  `return cqDirectoryGW.managingOrganization === 'Epic';`